akka-actor-tests/src/test/scala/akka/io/TcpConnectionSpec.scala

```diff
@@ -364,7 +364,7 @@ class TcpConnectionSpec extends AkkaSpec("""
         ConfigFactory.load(
           ConfigFactory.parseString("akka.io.tcp.direct-buffer-size = 1k")
             .withFallback(AkkaSpec.testConf))
-      override implicit def system: ActorSystem = ActorSystem("respectPullModeTest", config)
+      override lazy val system: ActorSystem = ActorSystem("respectPullModeTest", config)
```

Comment:
Isn't a val just fine? I think the purpose here is to have a system with different configuration than the rest. I guess this can be also moved to a different Spec if necessary.